### ISSUE TITLE
Checking maintainer trust fails with "Stream error in the HTTP/2 framing layer"

### ISSUE THREAD

**Issue by WieeRd** [alexheretic/aurto#98]
```
❯ aurto --version
aurto v0.12.9: simple management tool for the 'aurto' repository
```
```
❯ aurto add {any aur package}
aurto: Checking maintainer trust...
Error: Error { description: "Stream error in the HTTP/2 framing layer", code: 92, extra: Some("HTTP/2 stream 1 was not closed cleanly: PROTOCOL_ERROR (err 1)") }
```

Aurto was working fine last week, and today I tried to use it and this happened.
It gets stuck on 'Checking maintainer trust' for a long time, and eventually outputs the error above.

I couldn't collect any further information on what's going on, since I didn't know where to look.
What should I check?

**Comment by alexheretic** [comment]
It looks like the aur.archlinux.org/rpc API aurto has been using has changed and so no longer works. We'll need to update to use whatever it has changed to.

**Comment by alexheretic** [comment]
Fixed aur version `0.12.10-2`. Upgrade manually with something like:
```
aur sync --database=aurto --makepkg-conf=/etc/aurto/makepkg-chroot.conf --pacman-conf=/etc/aurto/pacman-chroot.conf --chroot --no-view aurto
```

**Comment by WieeRd** [comment]
Can confirm it fixed the issue :+1:
